### ISSUE TITLE
Node not appearing in Palette Library

### ISSUE THREAD

**Issue by pmknowles** [tnussbaum/node-red-contrib-eufy-home#1]
Your node does't seem to be available to load either from within Node-Red or in the Node-Red online library. Am I doing something wrong?

**Comment by tnussbaum** [comment]
You are correct. :)  I never made it that far in the progress of this library.  I made it for my own use, and I always meant to come back and finalize it.

If you are comfortable with git and the command line, you can do roughly the following steps to set it up and test it out:

- checkout the source code from github using the usual tools
- run npm install from the within the checked out source code directory to install the necessary dependencies
- navigate to the directory where node-red is installed (typically &lt;your home directory&gt;/.node-red)
- from within that directory, execute npm install &lt;directory where you checked out the source code&gt;
- restart node-red

This process will setup the source code to run with its dependencies, and then symbolically link it to the location where node-red would expect to find it. After the restart, it should be available in the palette of node-red

Good luck, and feel free to reply here if you have issues.  I'll do my best to assist.

**Comment by pmknowles** [comment]
Thanks I’ll give it a go.

Regards

Phil

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
